package.json

```diff
@@ -42,6 +42,7 @@
 		"ava": "^1.2.0",
 		"create-test-server": "^2.1.1",
 		"delay": "^4.1.0",
+		"eslint": "^5.14.1",
```

Comment:
?

Reply:
removed, it was added before I do npm install :)